v2.2/debug-merge-logs.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Merge debug logs for all nodes
+summary: Learn the command for merging the collected debug logs from all nodes in your cluster.
+toc: true
+---
+
+The `debug merge-logs` [command](cockroach-commands.html) merges the logs for all nodes in the cluster collected using the [`debug zip`](https://www.cockroachlabs.com/docs/stable/debug-zip.html) command. This command helps in debugging cluster issues by making it easier to parse the logs for all nodes and locating an issue.
+
+{{site.data.alerts.callout_danger}}
+The file produced by `cockroach debug merge-log` can contain highly sensitive, unanonymized information, such as usernames, passwords, and possibly your table's data. You should share this data only with Cockroach Labs developers and only after determining the most secure method of delivery.
```

Comment:
I found this sort of confusing because it's going to be exactly as sensitive as the log files that go in to it. Maybe change the wording of this callout to be about logs in general rather than about merge-logs specifically

Reply:
We do have a corresponding callout in the `debug zip` doc as well.